### ISSUE TITLE
memoria

### ISSUE THREAD

**Issue by crguezl** [ULL-ESIT-GRADOII-TFG/ghedsh#41]
Hola @alu0100816167 

¿Mejor hacer la memoria con la plantilla LaTeX?

Como tu lo veas, si prefieres word o google docs también está bien.  

Si es LaTeX puedes partir de la memo de Samuel:  https://github.com/ULL-ESIT-GRADOII-TFG/memoria-tfg-samuel

en ese caso, si quieres, quizá te facilite la vida abrir una cuenta en sharelatex  que es un latex en la nube para compartir:

https://es.sharelatex.com/

Por cierto, en la memoria debería haber un capítulo: 

"instrucciones para el desarrollador" 

que le indicara a los futuros TFGs como extender la versión actual con nuevas funcionalidades.

**Comment by alu0100816167** [comment]
> ¿Mejor hacer la memoria con la plantilla LaTeX?

Sí, no hay problema. Puedo partir de la de Samuel para redactar la mía.

> Por cierto, en la memoria debería haber un capítulo:
"instrucciones para el desarrollador"
que le indicara a los futuros TFGs como extender la versión actual con nuevas funcionalidades.

Vale, perfecto.
